shell/scl_source

```diff
@@ -66,7 +66,7 @@ if [ $_recursion == "false" ]; then
         else
             export X_SCLS="${_scls[$_i]} $X_SCLS"
         fi;
-        _i=$(($_i+1))
+        _i=$(("$_i"+1))
```

Comment:
why double brackets?

Reply:
That's bash syntax for numeric expressions?

Reply:
Ah, here should be just `_i=$(( _i + 1 ))`.